### ISSUE TITLE
Orchard.Layouts uses Details Display

### ISSUE THREAD

**Issue by Yaevh** [OrchardCMS/Orchard#7092]

**Comment by sebastienros** [comment]
Can you confirm you mean it happens on the front end?

**Comment by Yaevh** [comment]
Yes, the most annoying part is that all od that gets carried over to the front end and the reader is served a page with a title along the lines of " Documents to download - Document 5" instead of just "Documents to download" when there are four other documents on that page.
